portals/admin/src/main/webapp/site/public/locales/en.json

```diff
@@ -13,9 +13,10 @@
   "Admin.Throttling.Advanced.Throttling.policy.table.header.quota": "Quota",
   "Admin.Throttling.Advanced.Throttling.policy.table.header.quota.policy": "Quota Policy",
   "Admin.Throttling.Advanced.Throttling.policy.table.header.unit.time": "Unit Time",
+  "Admin.Throttling.Application.Throttling.Policy.add.burst.limits.details": "Burst Control (Rate Limiting)",
   "Admin.Throttling.Application.Throttling.Policy.add.data.amount.helper.text": "Bandwidth allowed",
   "Admin.Throttling.Application.Throttling.Policy.add.description.helper.text": "Description of the throttle policy",
-  "Admin.Throttling.Application.Throttling.Policy.add.general.details": "General Details",
+  "Admin.Throttling.Application.Throttling.Policy.add.general.details": "General Detailser",
```

Comment:
Any reason to change this String?

Reply:
Fixed in https://github.com/wso2/apim-apps/pull/503/commits/883bfab1a5cfa24b70df36212b59aa5b98b59ecb